.changeset/three-carrots-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components-development-kitchen': minor
```

Comment:
```suggestion
'@guardian/source-react-components-development-kitchen': patch
```
this should be a patch, but only because we havn't worked out how to release this properly yet (in proper semver terms this is a `minor` change, but this package can't really follow semver since any change could be breaking at any point). we should resolve this somehow